model/task.graphql

```diff
@@ -1,15 +1,48 @@
-""" 
+"""
 @model
 """
 type Task {
-  id: ID!
+  """
+  @db.primary
+  """
+  _id: ObjectID!
   title: String!
   description: String!
   status: TaskStatus
+  type: String
+  priority: Int
+  public: Boolean
+  status: TaskStatus
+  startDate: Date
+  payload: JSON
+
+  """
+  @oneToMany field: 'note'
+  """
+  comments: [Comment]!
+}
+
+"""
+@model
+@crud.delete: false
+"""
+type Comment {
+  """
+  @db.primary
+  """
+  _id: ObjectID!
+  message: String!
+  author: String!
 }
 
 enum TaskStatus {
   OPEN
   ASSIGNED
   COMPLETE
-}
\ No newline at end of file
+}
+
+### Custom types used by model
```

Comment:
Those are the examples of the Scalars we should have added out of the box.

Reply:
Yeap.